website/docs/Installation.md

```diff
@@ -37,6 +37,11 @@ pip install flaml[vw]
 pip install flaml[forecast]
 ```
 
+* natural language processing
```

Comment:
please mention what learners are they, like in the line of time series forecaster.